### ISSUE TITLE
[Bug]: invalid literal for int() with base 10: 'null'

### ISSUE THREAD

**Issue by JiehangXie** [milvus-io/milvus#25491]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.2.11
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.13
- OS(Ubuntu or CentOS): Windows
- Attu: 2.2.7
```

### Current Behavior

I edited milvus standalone version in `docker-compose.yml`, upgrading from 2.2.3 to 2.2.11 directly.
After upgrading, all collections created by milvus 2.2.3 is running normally.

Then I created a new collection by Attu 2.2.7, when connected by PyMilvus 2.2.13, it occured an error like this:
![5777f0cc8ee0718905c6ff31a731bb2](https://github.com/milvus-io/milvus/assets/51190264/c0004d28-4968-401d-8b6f-6f2856307ecf)

And then I creeated a new collection by Pymilvus API, and it works.
![9db14f781715673f9e642a784b89be7](https://github.com/milvus-io/milvus/assets/51190264/8496b6da-423f-4873-87d2-756bf23d3128)

**Comment by yanliang567** [comment]
sounds like an Attu issue? 
/assign @shanghaikid 
/unassign

**Comment by shanghaikid** [comment]
can you show me the schema of the collection that you created from attu?

**Comment by JiehangXie** [comment]
> can you show me the schema of the collection that you created from attu?

![image](https://github.com/milvus-io/milvus/assets/51190264/927c2537-692d-4fba-b0c2-65f941f2454e)
![image](https://github.com/milvus-io/milvus/assets/51190264/2811d4df-1d14-4821-8adf-aefb53ac701a)

**Comment by shanghaikid** [comment]
Seems pymilvus starts to check param type in the latest version, Node doesn't support int64, attu should format some data to number before passing to milvus. will fix it in later version of attu.

**Comment by shanghaikid** [comment]
attu will fix this. and milvus will also do some check here.

**Comment by yanliang567** [comment]
closed as verified by #25879
